corehq/apps/app_manager/models.py

```diff
@@ -4668,6 +4668,9 @@ def make_reversion_to_copy(self, copy):
 
         return app
 
+    def profile_url(self):
```

Comment:
Or in other words I think looking at the other commit you referenced, this is missing a `@property`?